src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -494,7 +494,7 @@ Tap and hold to open RVX settings."</string>
     <string name="revanced_disable_speed_overlay_summary">"Disable 'Playing at 2x speed' while holding down.
```

Comment:
I don't know how to change `Disable 'Playing at 2x speed' while holding down.`.

For me, the overlay just says `2x>>` instead of `Playing at 2x speed`. 

Does it make sense to rename this option to something like `Disable hold to 2x speed`? This emphasizes that this option doesn't just hide an overlay but disables this action entirely.

Reply:
When this was first added to YouTube by Google, the `Playing at 2x speed` label was used. (Added from YouTube 18.19.36)
At some point, it seems to have been changed to `2x >>` label.